src/main/java/com/tabnine/binary/fetch/BootstrapperSupport.java

```diff
@@ -0,0 +1,73 @@
+package com.tabnine.binary.fetch;
```

Comment:
Add private/public access levels to all methods and members. No need to make anything package-protected.